devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/tooling/dockerfiles/codestart.yml

```diff
@@ -2,3 +2,17 @@
 name: dockerfiles
 type: tooling
 preselected: true
+language:
+  base:
+    data:
+      dockerfile:
+        jvm:
+          from: registry.access.redhat.com/ubi8/ubi-minimal:8.1
+          java-package: java-11-openjdk-headless
+          run-java-version: 1.3.8
```

Comment:
what is that flag actually used for ?

Reply:
This is not "used", just we have it all in one location